source/extensions/filters/http/oauth2/filter.h

```diff
@@ -40,28 +40,28 @@ class SecretReader {
 public:
   virtual ~SecretReader() = default;
   virtual const std::string& clientSecret() const PURE;
-  virtual const std::string& tokenSecret() const PURE;
+  virtual const std::string& hmacSecret() const PURE;
```

Comment:
why this naming change is necessary?

Reply:
It's a small refacor: rename variables related to token_secret to hmac_secret to improve clarity and consistency.

This value of this variable is extensions.filters.http.oauth2.v3.OAuth2Credentials.hmac_secret, but it's named tokenSecret, which is very confusing.

https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/oauth2/v3/oauth.proto#envoy-v3-api-msg-extensions-filters-http-oauth2-v3-oauth2credentials